scripts/scp-deploy-frontend.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 # Deploy the frontend to the glassfish home directory and run bower
-export SERVER=vm
+export SERVER=instance
```

Comment:
@SirOibaf do we need to commit this file?

Reply:
We are not going to.